src/interval_heap.rs

```diff
@@ -399,7 +407,6 @@ mod test {
             let mut ih = IntervalHeap::from_vec(tmp);
             for _ in range(0, 100) {
                 ih.push(rng.next_u32());
-                assert!(is_interval_heap(as_slice(&ih)));
```

Comment:
Good question. Per [crates.io] (http://doc.crates.io/manifest.html#the-[profile.*]-sections), the test profile uses `cfg(ndebug)` by default. It's possible to disable in `Cargo.toml`, I suppose, but I don't know what the rationale would be, and I doubt you'd merge such a change to the project.